dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides

```diff
@@ -0,0 +1,58 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+
+WS-TraceGroup: WSSECURITY
+
+cxfVersion=2.6.2
+
+Bundle-Name: com.ibm.ws.org.apache.cxf.ws.security.2.6.2
+Bundle-Description: cxf ws security, version 2.6.2
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.ws.security.2.6.2
+Bundle-ActivationPolicy: lazy
+
+
+Export-Package: \
```

Comment:
This is lacking a version number. You should look at the original version of the org.apache.cxf.ws.security.2.6.2/META-INF/Manifest.mf file and make sure that the packages listed under it's Export-Package header match what's being exported in the Export-Package header of this bnd.overrides file. Do not use a org.apache.cxf.ws.security.* wild card, rather explicitly list each individual package with it's version number also listed.